ppr-ui/src/components/tables/common/TableRow.vue

```diff
@@ -557,8 +564,9 @@ export default defineComponent({
     }
 
     const isEnabledMhr = (item: MhRegistrationSummaryIF) => {
-      return item.statusType === APIStatusTypes.MHR_ACTIVE && isRoleQualifiedSupplier.value &&
-        (item.registrationDescription === APIMhrDescriptionTypes.REGISTER_NEW_UNIT ||
+      return item.statusType === APIStatusTypes.MHR_ACTIVE && (isRoleQualifiedSupplier.value &&
```

Comment:
To make it cleaner, maybe we can put all roles into a separate computed property?

Reply:
Also, doesn't `isRoleStaff` sort of cover all roles but `helpdesk`?
It might even cover that one too, not sure what a helpdesk account looks like but they might have staff role as well.

Reply:
If not all good but worth looking into.

Reply:
Ok, thank you for checking. 
Hopefully we can revisit/clean up in the future.